flow/table.go

```diff
@@ -113,29 +120,280 @@ func updateTCPFlagTime(prevFlagTime int64, currFlagTime int64) int64 {
 	return currFlagTime
 }
 
+func kernFlowKeyOuter(kernFlow *C.struct_flow) string {
```

Comment:
please keep eBPF probe code in the probe itself

Reply:
EBPFFlow is now part of the table isn't it?